java/src/jmri/jmrit/ussctc/RouteLock.java

```diff
@@ -116,5 +116,13 @@ public boolean isLockClear(LockLogger lockLogger) {
     boolean isSignalClear(@Nonnull NamedBeanHandle<SignalHead> handle) {
         return handle.getBean().getState() != SignalHead.RED;
     }
+
+    @Override
+    public String toString() {
+        String retval = isLockClear(debugLockLogger) ? "clear " : "locked";
+        retval = retval+debugLockLogger.memory.getValue();
+        return retval;
+    }
```

Comment:
This method seems to be the same in all the implementing classes. It seems to be a good idea to move it to the interface Lock, if possible.

Another question is that "clear " has a space after clear. But there is no space after locked. Is this intended? Maybe change it to:
`retval = retval + " " + debugLockLogger.memory.getValue();`

Reply:
`Lock` is an interface. It can't hold method implementations unless they're `default`, and you can't override `Object.toString()` with a default method.